### PR TITLE
fixed typo in serverless package mentioned in issue #63402

### LINKED ISSUE #63402: Typo, 'mehtod' should be 'method'
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/014a3135cc52eba237576ea834b909c486714fe4/types/serverless/plugins/aws/package/compile/events/apiGateway/lib/validate.d.ts#L9

### PR DESCRIPTION
**Pull request by defnot001**
If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: [https://github.com/DefinitelyTyped/DefinitelyTyped/issues/63402]

Fixes #63402

### CODE REVIEW

**Review by fargito — APPROVED** [review]

**Comment by defnot001** [comment]
> Hey @defnot001,
> 
> 😒 Your PR doesn't modify any tests, so it's hard to know what's being fixed, and your changes might regress in the future. Please consider [adding tests](https://github.com/DefinitelyTyped/DefinitelyTyped#user-content-test-editing-an-existing-package) to cover the change you're making. Including tests allows this PR to be merged by yourself and the owners of this module.
> 
> _**This can potentially save days of time for you!**_

Sorry, I am new to contributing :). I figured that in this specific case where it's just chaging one word, it doesn't make sense to add new tests. Is that correct?

**Comment by fargito** [comment]
Hello @defnot001, thank you for your contribution, could you split it in 2 PRs (one for each package)? This will make the review process easier

**Comment by defnot001** [comment]
Hello and thanks for the suggestion.
#Unfortunately I didn't realize this would add the second change to the first PR. Still new to contributing. I will try my best. ☺️

**Comment by defnot001** [comment]
@fargito I think this should be good now, can you check again please?

**Review by fargito — APPROVED** [review]

**Comment by defnot001** [comment]
I changed the function to use the interface that was already there but never referenced. Also added tests now that i actually changed more than a typo :).

File: types/serverless/serverless-tests.ts [hunk 1]
```diff
@@ -128,11 +128,27 @@ getHttp(
...
     'myFunction',
 );
+
+// @ts-expect-error - Test missing properties
+getHttp({}, 'myFunction');
+
+// @ts-expect-error - Test invalid properties
+getHttp({}, 'myFunction', { invalid: 'property' });
+
+getHttp({
```
Comment (sandersn) [inline types/serverless/serverless-tests.ts:145]: you should add a test of the return type for getHttp. Dropping the type inference is probably fine but means, technically, that if you save the result in a variable `x` then `x.method` will be available now, but not in the old code.

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/serverless/serverless-tests.ts (hunk 1 above)
Comment (defnot001) [reply types/serverless/serverless-tests.ts:145]: Very good point, will do soon :). Thanks a lot!

**Comment by defnot001** [comment]
Okay so there is a bigger problem here. I installed the serverless npm package in a fresh TS project and whenever I try to import the getHttp function from the path that was specified in the test here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6f518eab7a19abf734c70e7c4589250554ec678b/types/serverless/serverless-tests.ts#L4 and try to log the function, I get a "cannot find module" error. 

I assume that the serverless devs changed their file structure somewhere down the line because when I import it from the correct location, it works.

```ts
import { getHttp } from 'serverless/lib/plugins/aws/package/compile/events/api-gateway/lib/validate';
```

I feel like this requires a lot more work but unfortunately this PR is on my master branch so i'd like to close it and work on a fix in my branch.

**Pull request closed without merging**